src/AutoRest.CSharp/Mgmt/AutoRest/MgmtConfiguration.cs

```diff
@@ -105,6 +106,7 @@ public void Write(Utf8JsonWriter writer, string settingName)
             IsArmCore = !IsValidJsonElement(armCore) ? false : Convert.ToBoolean(armCore.ToString());
             DoesResourceModelRequireType = !IsValidJsonElement(resourceModelRequiresType) ? true : Convert.ToBoolean(resourceModelRequiresType.ToString());
             DoesResourceModelRequireName = !IsValidJsonElement(resourceModelRequiresName) ? true : Convert.ToBoolean(resourceModelRequiresName.ToString());
+            DoesSingletonRequiresKeyword = !IsValidJsonElement(singletonRequiresKeyword) ? false : Convert.ToBoolean(singletonRequiresKeyword.ToString());
```

Comment:
make name consistent?

Reply:
Local variables I believe we do not need to make them consistent. If you are referring to the property names, it is consistent.

Reply:
And this configuration is here to keep backward compatibility